### ISSUE TITLE
Check Node18 compatibility

### ISSUE THREAD

**Issue by Cafe137** [ethersphere/swarm-cli#370]
As reported by nugaon the app does not work with the newest version

**Comment by Cafe137** [comment]
```
/Users/aron/Code/swarm-cli/dist/index.js:3529
                    var t = !!r.ActiveXObject || "-ms-scroll-limit" in document.documentElement.style && "-ms-ime-align" in document.documentElement.style,
                                                                       ^

ReferenceError: document is not defined
    at w (/Users/aron/Code/swarm-cli/dist/index.js:3529:72)
    at Object.n (/Users/aron/Code/swarm-cli/dist/index.js:3552:24)
    at /Users/aron/Code/swarm-cli/dist/index.js:3618:24
    at 7178 (/Users/aron/Code/swarm-cli/dist/index.js:3619:16)
    at n (/Users/aron/Code/swarm-cli/dist/index.js:15626:25)
    at 7365 (/Users/aron/Code/swarm-cli/dist/index.js:5644:27)
    at n (/Users/aron/Code/swarm-cli/dist/index.js:15626:25)
    at 4161 (/Users/aron/Code/swarm-cli/dist/index.js:12105:28)
    at n (/Users/aron/Code/swarm-cli/dist/index.js:15626:25)
    at 9068 (/Users/aron/Code/swarm-cli/dist/index.js:11364:23)
```

**Comment by Cafe137** [comment]
```
ReferenceError: document is not defined
    at w (/Users/aron/Code/swarm-cli/node_modules/@ethersphere/manifest-js/dist/webpack:/ManifestJs/node_modules/@ethersphere/bee-js/dist/index.browser.min.js:2:7993)
```

With node.js 18 it seems to the browser bundle is used

**Comment by Cafe137** [comment]
```
/Users/aron/Code/swarm-cli/node_modules/@ethersphere/manifest-js/dist/webpack:/ManifestJs/node_modules/blob-polyfill/Blob.js:519
				"-ms-scroll-limit" in document.documentElement.style &&
                          ^
ReferenceError: document is not defined
    at fixFileAndXHR (/Users/aron/Code/swarm-cli/node_modules/@ethersphere/manifest-js/dist/webpack:/ManifestJs/node_modules/blob-polyfill/Blob.js:519:27)
```

**Comment by Cafe137** [comment]
Will be fixed by #371

**Comment by Cafe137** [comment]
Tracking in https://github.com/ethersphere/swarm-cli/issues/387
